setup.py

```diff
@@ -9,10 +9,10 @@
 
 
 name = 'dsnparse'
-with open("{}.py".format(name), encoding='utf-8') as f:
+with open("{name}.py".format(name=name), encoding='utf-8') as f:
     version = re.search("^__version__\s*=\s*[\'\"]([^\'\"]+)", f.read(), flags=re.I | re.M).group(1)
 
-with open('README.rst', encoding='utf-8') as f:
+with open('README.md', encoding='utf-8') as f:
```

Comment:
Did you have trouble running `setup.py`? I think the answer is a manifest file, my workflow involves running:

```
$ pandoc --from=markdown --to=rst --output=README.rst README.md
```

before using `setup.py` in any way, so this is a blind spot for me, using the `README.md` file won't render on pypi though, so we should fix this another way